docs/src/main/sphinx/develop/tests.md

```diff
@@ -0,0 +1,424 @@
+# Testing guidelines
+
+The Trino project aims to provide high quality artifacts and features to all
```

Comment:
There is a small paragraph about testing in the connector developer guide, maybe it should link here?

Reply:
Where is that? Also .. do you think we need more in the overview page?

Reply:
I can't find it, I think I mixed something up, please disregard this comment.